package.json

```diff
@@ -60,7 +61,6 @@
     "debug": "node --nolazy --inspect --debug-brk scripts/kibana --dev",
     "precommit": "node scripts/precommit_hook",
     "karma": "karma start",
-    "elasticsearch": "grunt esvm:dev:keepalive",
```

Comment:
I'd prefer that we didn't get rid of the elasticsearch task since `yarn es snapshot` is just a new thing to learn that doesn't really buy us anything, but if we do decide to get rid of it we should should at least replace it with something like `echo 'use `yarn es snapshot`'`.

Reply:
Good call, unless `yarn es snapshot` is modified to read Kibana's package.json people will need to send the right version number for the branch. maybe could be the difference between `yarn es` and `yarn elasticsearch`? (I still wish this was just `node scripts/es`)

Reply:
This is pretty much how it works currently. `scripts/es.js` [sets defaults, like the version](https://github.com/elastic/kibana/pull/17168/files#diff-4f195c99d66a4406b12920685f9f364fR13) to be consistent with that in the `package.json`. In addition to the version, we also default the path to the ES source. This results in using the nightly for the version you are on.

Reply:
@tylersmalley, so is [getBranch](https://github.com/elastic/kibana/pull/17168/files#diff-bb452c917bcb128228235121314111d2R6) used anymore?  It was formerly used in the esvm config file to figure out which branch to download.

Reply:
@trevan good catch, I have removed that here and updated the last reference of it to use the package directly. d8bdf257dad05565b8cc0fd848112c3495c2cd33